src/components/exchange/ExchangeFloatingPanels.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { View } from 'react-native';
+import { Box, Stack } from '@rainbow-me/design-system';
+
+const ExchangeFloatingPanels = React.forwardRef<View>((props, ref) => {
+  const children = props.children!;
+  return (
+    <Box ref={ref}>
+      <Stack {...props} children={children} />
```

Comment:
Is the type of `props` valid to spread here? This changes the type of `ExchangeFloatingPanels` to be a superset of `Stack` from the DS rather than Column.

Reply:
I could be wrong but from looking at the code the `Column` component is a `Flex` with `flexDirection="column"` set to it so `Stack` like the closest equivalent in the new DS.

Reply:
The previous implementation had `pointerEvents="box-none"` – do we need to have this here too?

Reply:
I also only see this component being used in `AnimatedExchangeFloatingPanels`. Do we need to extract it into a separate file here?

Reply:
Another note: I don't believe the consumer of this component even passes any props (see https://github.com/rainbow-me/rainbow/pull/4014/files#diff-bcc39e0a9afb8c2146a721fb287948c2c1835246c62b2509304194f144308082R901). Do we need to spread?

Reply:
Yeah good point, ive just removed all spreading and explicitly only passed the `children`